### PR TITLE
ci: adds script for writing schemas

### LINKED ISSUE #11: Publish schemas to s3
**Is your feature request related to a problem? Please describe.**
As a user, I'd like to pull the data models from a database

**Describe the solution you'd like**
- [ ] A github action will convert the csv files into json and push the json into docdb.
- [ ] On document record per csv file name like `modalities`, `harp_types`, etc.
- [ ] It will check if the record exists and overwrite the existing record
```
{"_id": UUID, "model": "modalities", "contents":[list of models] }
```

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by yosefmaru**
Closes #11

### CODE REVIEW

File: .github/workflows/publish_models_dev.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          role-session-name: github-schema-upload-session
+          aws-region: ${{ env.AWS_DEFAULT_REGION }}
+      - name: Install dependencies
+        run: | 
+          python -m pip install -e .
+          python -m pip install aind-data-access-api[full]
+      - name: Publish models to docdb
+        run: | 
+          python -m pip install -e .
+          python -m pip install aind-data-access-api[full]
```
Comment (jtyoung84) [inline .github/workflows/publish_models_dev.yml:43]: I think you only [docdb] instead of [full]?

File: .github/workflows/test_and_lint.yml [hunk 2]
```diff
@@ -20,6 +20,7 @@ jobs:
     - name: Install dependencies
       run: | 
         python -m pip install -e .[dev]
+        python -m pip install aind-data-access-api[full]
```
Comment (jtyoung84) [inline .github/workflows/test_and_lint.yml:23]: You shouldn't need this?

File: src/aind_data_schema_models/scripts/write_to_docdb.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    """
+    Writes models to docdb
+    """
+    for file_name in os.listdir(folder_path):
+        if file_name.endswith('.csv'):
+            collection = file_name[:-4]
+            credentials.collection = collection
+            csv_file_path = os.path.join(folder_path, file_name)
+            json_data = csv_to_json(csv_file_path)
+            with DocumentDbSSHClient(credentials=credentials) as doc_db_client:
```
Comment (jtyoung84) [inline src/aind_data_schema_models/scripts/write_to_docdb.py:38]: Isn't there a way to avoid constructing this in the for loop?

**Review by jtyoung84 — APPROVED** [review]

File: .github/workflows/publish_models_dev.yml [hunk 4]
```diff
@@ -0,0 +1,41 @@
+name: Publish models to dev
+on:
+  push:
+    branches:
+      - dev
```
Comment (jtyoung84) [inline .github/workflows/publish_models_dev.yml:5]: I don't think there is a dev branch on this repo

Comment (yosefmaru) [reply .github/workflows/publish_models_dev.yml]: Thanks for the catch. Switched dev workflow back to main branch.

**Review by jtyoung84 — APPROVED** [review]

**Pull request merged**